### ISSUE TITLE
Publish OpenTelemetry schema for each spec version

### ISSUE THREAD

**Issue by tigrannajaryan** [open-telemetry/opentelemetry-specification#1693]
According to [OTEP 0152](https://github.com/open-telemetry/oteps/blob/main/text/0152-telemetry-schemas.md) we need to publish a schema file per version.

Here is the preliminary plan to do it:

- In the https://github.com/open-telemetry/opentelemetry-specification repo we will put the source files in some directory, e.g. `/schemas`, so `/schemas/1.4.0` will be a YAML file containing the schema for specification version 1.4.0.
- In https://github.com/open-telemetry/opentelemetry.io repository we will have all schema files in `/static/schemas/`.
- When a new version of the specification is planned to be released the corresponding new YAML file should be already created https://github.com/open-telemetry/opentelemetry-specification repo, under the `/schemas` directory. This is a pre-requisite for spec release (and we can automate verification of this pre-requisite in the future).
- The https://github.com/open-telemetry/opentelemetry-specification repo will have a manually triggered github action that creates a new commit from `/schemas` and pushes it to https://github.com/open-telemetry/opentelemetry.io repo as a new PR (similar to what we already have for [Java](https://github.com/open-telemetry/opentelemetry-java/blob/main/.github/workflows/docs-update.yml)).
- Immediately after a new specification release the github action will be triggered manually by the person who does the spec release, as a result a PR will be created in https://github.com/open-telemetry/opentelemetry.io repo and once the PR is merged the new schema file will become available at `https://opentelemetry.io/schemas/<version>` (deployed via Netlify).
- In the future we can automate the process a bit more so that the PR is created automatically when the spec release is made without requiring manual triggering, and possibly also merge the PR automatically if it passes the build.

Corresponding discussion in the website repo: https://github.com/open-telemetry/opentelemetry.io/issues/577

**Comment by tigrannajaryan** [comment]
Draft PR open for manually publishing the first version https://github.com/open-telemetry/opentelemetry.io/pull/599
